.env.example

```
@@ -53,4 +53,15 @@ NODE_ENV="development"
 #GITHUB_API_KEY="your_github_api_key_here"
 
 MASTRA_BASE_URL="https://localhost:4111"
+
+# Optional
 JWT_TOKEN="your_jwt_token_here"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)